L1Trigger/L1CaloTrigger/python/test_L1EGCrystalClusterEmulator_cfg.py

```diff
@@ -0,0 +1,76 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process("L1AlgoTest",eras.Phase2_trigger)
```

Comment:
please use the actual era for the phase 2 workflow being tested
(phase2_trigger is an internal use modifier, Phase2_trigger with that capitalization is not anything)

Reply:
this is relic from previous versions of the integration branch

In 11_0_0_pre6  the l1 integration branch The driver config file which was used for testing uses the Phase2 era
https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-CMSSW_11_1_0_pre6/L1Trigger/L1TCommon/test/reprocess_L1TrackTrigger_L1T.py#L10

Reply:
"please use the actual era for the phase 2 workflow being tested" does not mean "use the Phase2 era"

different scenarios have different associated eras, seen in https://github.com/cms-sw/cmssw/blob/f18f5a71e7278be55d457d17e38379b11cd04c19/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L740